### ISSUE TITLE
Navigator mouse is not enabled after disabling it

### ISSUE THREAD

**Issue by sapkmm** [openseadragon/openseadragon#1306]
Navigator mouse is not enabled after disabling the it. Navigator mouse interaction is disabled using navigator.setMouseNavEnabled(false) and enabled using navigator.setMouseNavEnabled(true);

However this is not working mouse is disabled properly, but enabling doesn't seems to work.

https://codepen.io/anon/pen/vJoRmp

**Comment by iangilman** [comment]
Good catch! Looks like this probably happens because while the navigator inherits `setMouseEnabled` from the Viewer class (the navigator is a subclass of the viewer class), it does different things with its mouse tracker (it kills the inherited innerTracker and creates its own, and seems to just ignore its inherited outerTracker). 

Probably the fix is to give the navigator class its own `setMouseEnabled` that just disables the innerTracker and doesn't do anything with the outerTracker. 

Would you be up for making this fix? I can point you to the right parts of the code if you'd like.

**Comment by sapkmm** [comment]
Sure, point me to the exact code and changes required, I'll try to do it.

**Comment by iangilman** [comment]
Great! Basically you want to copy setMouseNavEnabled from Viewer: 

https://github.com/openseadragon/openseadragon/blob/master/src/viewer.js#L791

...into the Navigator's prototype, maybe around here: 

https://github.com/openseadragon/openseadragon/blob/master/src/navigator.js#L367

...but modify it so it ignores the outerTracker. In your copy you should probably remove all of the doc comments as well, since the function and event are already documented at the Viewer level and we don't need the doc duplication. 

Sound good?

**Comment by iangilman** [comment]
To be honest, I'm not entirely sure that my proposed fix will address the issue, but I figure it's a good start and may help get to the bottom of it. Once Navigator has its own setMouseNavEnabled, it can be tweaked to get the proper result.
